src/test/resources/GenerateTest/buf generate fails with no buf gen template file/build.gradle

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Is this file still used?

Reply:
Oops, good catch, this is no longer needed.